CompetitionMars/CompetitionMars/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,317 @@
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using CompetitionMars.DataModel;
+using CompetitionMars.Pages;
+using CompetitionMars.Utilities;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CompetitionMars.Tests
+{
+    [TestFixture]
+    public class CertificationTests : CommonDriver
+    {
+        private LoginPage loginPageObject = new LoginPage();
+        private CertificationPage certificationPageObject = new CertificationPage();
+        private JsonHelper jsonHelperObject = new JsonHelper();
+        private List<Certification> testData = new List<Certification>();
+        private ExtentReports extent;
+        private ExtentTest test;
+
+        [OneTimeSetUp]
+        public void SetupReport()
+        {
+            string reportPath = "C:\\internship notes\\CompetitionMars\\CompetitionMars\\CompetitionMars\\CompetitionMars\\Utilities\\ExtentReport\\BaseReport.cs";
+            ExtentHtmlReporter htmlReporter = new ExtentHtmlReporter(reportPath);
+            extent = new ExtentReports();
+            extent.AttachReporter(htmlReporter);
+        }
+
+        [SetUp]
+        public void SetUpActions()
+        {
```

Comment:
see the comment below around moving this to common place, you only need one  of this not for every test